sdk/src/common/BUILD

```diff
@@ -27,6 +27,18 @@ cc_library(
     include_prefix = "src/common",
     deps = [
         "//api",
+        "//sdk:headers",
         "//sdk/src/common/platform:fork",
     ],
 )
+
+cc_library(
+    name = "global_log_handler",
+    srcs = [
+        "global_log_handler.cc",
+    ],
+    deps = [
+        "//api",
+        "//sdk:headers",
+    ],
+)
```

Comment:
I am not familiar with BAZEL, and I am confused by the build script here.

The bazel build defines two libraries, "random" and "global_log_handler", while the CMake build defines only one library, named "opentelemetry_common"

This is a pre existing issue, but how come file random.cc is compiled in library "random" with bazel, and "opentelemetry_common" with CMake ?

I would expect both build to produce the same libraries with the same name, and put the same *.cc code in the same *.so library.

Reply:
Agree, though not related to this PR, it would be good to have both cmake and bazel produce a consistent set of libraries.

Reply:
Agree too. We can raise another PR to produce a consistent set of libraries between bazel and cmake.There are more libraries names need be reviewed.